src/components/memory/actual-free/index.html

```diff
@@ -0,0 +1,37 @@
+<html>
+    <head>
+        <script src="../../../../dist/hedwig-main.umd.js"></script>
+    </head>
+    <body>
+        <script>
+            var mock = [
+                { 
+                    "memory_actual_free": {
+                        "time": "2018-11-24T18:58:21Z",
+                        "value": 100
+                    }
+                },
+                {
+                    "memory_actual_free": {
+                        "time": "2018-11-25T23:58:21Z",
+                        "value": 200
+                    }
+                },
+                {
+                    "memory_actual_free": {
+                        "time": "2018-11-26T02:58:21Z",
+                        "value": 500
+                    }
+                },
+                {
+                    "memory_actual_free": {
+                        "time": "2018-11-27T10:58:21Z",
+                        "value": 250
+                    }
+                }        
+            ];
+
+            document.write('<memory-actual-free data-height="400" data-width="1000" data-graph=' + JSON.stringify(mock) + '></memory-actual-free>');
```

Comment:
Looks like `data-unit` is missing. May not be too necessary as the default is being set.